src/audio/Kconfig

```diff
@@ -33,6 +33,15 @@ config COMP_CHAIN_DMA
 	  help
 	    Chain DMA support in hardware
 
+config COMP_IPCGTW
```

Comment:
COMP_IPC_GTW for the rest of us with prescription glasses?

Reply:
@serhiy-katsyuba-intel best to say ```CONFIG_IPC4_GATEWAY```  as this make its obvious for everyone its an IPC4 config for gateways.